css/datagrid/datagrid.css

```diff
@@ -128,13 +128,22 @@
     width: 100%;
     display: block;
     overflow: hidden;
+    border: none;
+    background-color: transparent;
+    cursor: pointer;
+    font-weight: bold;
+    font-size: 1em;
```

Comment:
And it should be directly on the button, I think, instead of on the td. Or at least only on the td that contain the button. Now, for example, all, even unsortable `HeaderCells` have a cursor pointer.